lib/tasks/failing_spec_detector/print_log.rake

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+namespace :failing_specs_detector do
+  desc 'Print all logs in console'
+  task :print_log do
+    puts "Failing specs detector:\n"
+    log_file_paths = Dir['failing_specs_detector_log_*.txt']
+    log_file_paths.each do |file_path|
+      file = File.open(file_path, 'r')
+      file_data = file.read
+      puts file_data
+      File.delete(file_path)
```

Comment:
If I understood it correctly it just print the log files one by one, without grouping the failed specs across different files for the same failure message, right?

Reply:
yes, that's right. still working on that. The formatter is instantiated every time we run rspec which we do for each job, still trying to figure out how to work with that

Reply:
Considering it I have moved the PR to draft.

I think that a good strategy could be to store the result as Json or YAML to the file and then elaborate the results with the second task